### ISSUE TITLE
Support localized control text for the quality selector button

### ISSUE THREAD

**Issue by yokuze** [silvermine/videojs-quality-selector#7]
Currently, the quality selector button itself does not have any text in its `aria-label` and `.vjs-control-text` element.

We'd like to have support for localized text in the button that opens the menu. It would likely be a string like "Quality Selector".

The text for the menu items themselves already support localization.
